QGL/BasicSequences/RB.py

```diff
@@ -119,20 +107,14 @@ def TwoQubitRB(q1, q2, seqs, showPlot=False, suffix=""):
 
 
 def SingleQubitRB_AC(qubit, seqs, showPlot=False):
```

Comment:
I guess it makes sense to have the option everywhere, this was an experiment. I'll update